### PR TITLE
No bypass

### PR DESCRIPTION
**Pull request by frompotenza**
## Related Issue(s)
  - Remove bridge bypass 

## Other Dependent PRs
  - new-state-estimation (AUV-2024)

## Changes Made
- Removed the bypass topics (bridge.py only publishes to the sensors' topics now)
- Fix coordinate system so that the sensors' coordinates are all NED

## Tests Done
- Basic missions work and the pose signs seem correct
- We should test more with the dependent branch tho

## Related Branches (other than one that will be merged)
- new-state-estimation (AUV-2024)

### CODE REVIEW

File: src/bridge.py [hunk 1]
```diff
@@ -5,10 +5,11 @@
 
 from auv_msgs.msg import ThrusterForces, DeadReckonReport
 from geometry_msgs.msg import PoseArray, Vector3, Quaternion, Pose
-from sbg_driver.msg import SbgImuData, SbgEkfQuat
+from sbg_driver.msg import SbgImuData, SbgEkfQuat, SbgImuStatus
```
Comment (antoinedang) [inline src/bridge.py]: don't think we need this message import

**Review by antoinedang — APPROVED** [review]

**Comment by MrMondrian** [comment]
Do we need to merge a branch in AUV-2024 simultaneously?

**Comment by frompotenza** [comment]
> Do we need to merge a branch in AUV-2024 simultaneously?

it would be better. That's why I want to run a complete mission before. It is a big merge for both sides

**Comment by MrMondrian** [comment]
All quaternions should be name q_{framea}_{frameb}. @antoinedang @frompotenza make sure you guys make these changes at some point. I wasn't gonna approve because of that

**Pull request merged**